.moban.cd/changelog.yml

```diff
@@ -1,6 +1,12 @@
 name: moban
 organisation: moremoban
 releases:
+- changes:
+  - action: Added
+    details:
+    - "`#234`: Define template parameters on the fly inside `targets` section"
+    date: 01.03.2019
```

Comment:
deleting two more spaces on the left hand side will show date and version in change log